### PR TITLE
Assets management documentation

### PR DESCRIPTION
**Pull request by stphnpt**
### Subject

Add documentation to explain how to manage assets for generating
SonataDashboardBundle css and js files (back and front).

NB:
npm WARN SonataDashboardBundle@0.0.1 No repository field.
npm WARN SonataDashboardBundle@0.0.1 No license field.

### To do

- [x] Update the documentation

### CODE REVIEW

**Comment by core23** [comment]
We don't need a changelog for documentation

**Comment by stphnpt** [comment]
Shall I delete it then ?

File: Resources/doc/reference/assets.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Assets
+======
+
+Assets are managed with gulp which require nodejs/npm and a global gulp install.
```
Comment (greg0ire) [inline Resources/doc/reference/assets.rst]: "requires"

File: Resources/doc/reference/assets.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+* stylesheets are generated from .scss files located in ``/Resources/assets_src/src/scss``
+* javascripts are generated from files located in ``/Resources/assets_src/src/js``
```
Comment (greg0ire) [inline Resources/doc/reference/assets.rst]: are they generated or just copied?

Comment (stphnpt) [inline Resources/doc/reference/assets.rst]: css and js assets are generated yes, not copied.

**Comment by greg0ire** [comment]
LGTM

File: Makefile [hunk 3]
```diff
@@ -2,7 +2,7 @@
 #
 # It's auto-generated by sonata-project/dev-kit package.
```
Comment (OskarStark) [inline Makefile]: this file is auto generated

how to handle it @Soullivaneuh

**Comment by OskarStark** [comment]
**Please do not merge this, until @Soullivaneuh answers!**

File: Makefile (hunk 3 above)
Comment (Soullivaneuh) [inline Makefile]: The `Makefile` is for dev and test of the bundle. This is not for production integration.

Why not just add this on the doc?

In a more global way, we should decide the way to go to manage assets on **each** bundle.

See https://github.com/sonata-project/dev-kit/issues/137 for example.

**Comment by Soullivaneuh** [comment]
Sorry for the auto-close. This is because we decide to work on `master` only for now.

Don't worry, to continue the work, simply reopen a new PR with the same branch but against `master` this time. :+1:

**Pull request closed without merging**